BLSQ/iaso
# IA-3763 IA-3769 entities details refactor + unsupported fields

- Entity details should display label instead of variable's name
- select_multiple unsupported in entities reference form

Related JIRA tickets : IA-3763, IA-3769

## Self proofreading checklist



## Changes

- use form descriptor to display label of the selected option
- support `select all that apply,` as it seems to be `select_multiple` type of fiels in xlsform
- enable multi select in form submission `querybuilder` (we never enabled it)
- relayout beneficiary detail page to avoid white space and table at the bottom
- enlarge beneficiary type dialog to allow lots selected options in dropdown

## How to test

- Use  this xls to create a form,
[fiche_individuelle_acteurs_2024120802.xlsx](https://github.com/user-attachments/files/18080161/fiche_individuelle_acteurs_2024120802.xlsx)
- Create a beneficiary type with this form as reference form, select "Thématiques des formations recues" , " Etat civil", "Prélever les coordonnées géographiques" to be visible in the list fields.
- Create a submission with the new form, make sure you answer previous questions
- Create an entity using the admin and link it to the submission
- Visit entity detail page in the dashboard. check that all fields are displayed correctly


## Print screen / video

![Screenshot 2024-12-10 at 14 15 20](https://github.com/user-attachments/assets/e45b8355-142a-4ced-82ae-c8ca6217ba0c)
<img width="1222" alt="Screenshot 2024-12-10 at 12 37 39" src="https://github.com/user-attachments/assets/bf157f87-e396-4a7b-a296-5421403b9952">


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
